### PR TITLE
Fixes for passing test-toolkit.

### PR DESCRIPTION
**Pull request by andrei-maroz**
[UI SandBox](https://portal.azure.com/?feature.customPortal=false&#blade/Microsoft_Azure_CreateUIDef/SandboxBlade)

### CODE REVIEW

**Review by joytandon17 — APPROVED** [review]

File: main/mainTemplate.json [hunk 1]
```diff
@@ -170,9 +170,9 @@
     "cloudConfig": "#cloud-config\nhost_setup:\n",
     "customData": "[concat(variables('cloudConfig'), '  access_portal: ', variables('hostSetup').accessPortal, '\n  jointoken: ', parameters('jointoken'), '\n  config_server: ', variables('hostSetup').configServer, '\n  notification_server: ', variables('hostSetup').notificationServer, '\n')]",
     "hostSetup": {
-      "accessPortal": "",
-      "configServer": "",
-      "notificationServer": ""
+      "accessPortal": "env-2a.test.infoblox.com",
+      "configServer": "cpnoa-env-2a.test.infoblox.com:443",
+      "notificationServer": "cpnoa-env-2a.test.infoblox.com:443"
     },
```
Comment (joytandon17) [inline main/mainTemplate.json:176]: Do we need to hardcode these values in release branch?

File: main/mainTemplate.json [hunk 2]
```diff
@@ -6,6 +6,10 @@
       "type": "string",
       "defaultValue": "[deployment().properties.templateLink.uri]"
     },
+    "_artifactsLocationSasToken": {
```
Comment (jkraj) [inline main/mainTemplate.json:9]: add  description

File: main/mainTemplate.json [hunk 3]
```diff
@@ -110,7 +109,7 @@
     },
     "virtualNetworkAddressPrefix": {
```
Comment (jkraj) [inline main/mainTemplate.json:110]: `prefix` means the portion after `/`.  the  correct name would be `virtualNetworkAddress` or `virtualNetworkAddresswithPrefix`

File: main/mainTemplate.json (hunk 2 above)
Comment (jkraj) [reply main/mainTemplate.json:9]: SasToken of what?

File: main/createUiDefinition.json [hunk 4]
```diff
@@ -65,7 +65,7 @@
         "toolTip": "Pass Jointoken.",
```
Comment (jkraj) [inline main/createUiDefinition.json:65]: Enter Jointoken obtained from CSP

File: main/createUiDefinition.json [hunk 5]
```diff
@@ -65,7 +65,7 @@
...
         "constraints": {
           "required": true,
-          "regex": "^[a-zA-Z0-9_-]{44}$",
+          "regex": "^[a-zA-Z0-9_-]{44,44}$",
           "validationMessage": "Insert validate Jointoken."
```
Comment (jkraj) [inline main/createUiDefinition.json:69]: should we say just `Invalid Jointoken` ?  or `Insert valid Jointoken`

File: main/createUiDefinition.json [hunk 6]
```diff
@@ -92,7 +92,7 @@
               "virtualNetwork": "Only virtual networks in the currently selected subscription and location are listed. If you choose to create a new virtual network, it will be created in the same subscription, location, and resource group as the storage account."
             },
             "defaultValue": {
-              "addressPrefixSize": "/8"
+              "addressPrefixSize": "/16"
```
Comment (jkraj) [inline main/createUiDefinition.json:95]:
```suggestion
              "addressPrefix": "/16"
```

File: main/createUiDefinition.json [hunk 7]
```diff
@@ -92,7 +92,7 @@
...
             },
             "constraints": {
               "minAddressPrefixSize": "/29"
```
Comment (jkraj) [inline main/createUiDefinition.json:98]: Based on what these constraint are set here? why can I use different one?

File: main/mainTemplate.json (hunk 2 above)
Comment (andrei-maroz) [reply main/mainTemplate.json:9]: Added description.
We do not use this parameter, but test-tool-kit checks it.

File: main/mainTemplate.json (hunk 3 above)
Comment (andrei-maroz) [reply main/mainTemplate.json:118]: This is addressPrefix for virtualNetwork.

File: main/createUiDefinition.json (hunk 4 above)
Comment (andrei-maroz) [reply main/createUiDefinition.json]: Updated.

File: main/createUiDefinition.json (hunk 5 above)
Comment (andrei-maroz) [reply main/createUiDefinition.json]: Good catch. Fixed typo.

File: main/createUiDefinition.json (hunk 6 above)
Comment (andrei-maroz) [reply main/createUiDefinition.json]: Removed default value as not needed more.

File: main/createUiDefinition.json (hunk 7 above)
Comment (andrei-maroz) [reply main/createUiDefinition.json:95]: Removed default value as not needed more.

File: main/mainTemplate.json [hunk 8]
```diff
@@ -110,7 +117,7 @@
...
       "type": "string",
-      "defaultValue": "10.0.0.0/8",
+      "defaultValue": "10.0.0.0/16",
```
Comment (joytandon17) [inline main/mainTemplate.json:120]: I think We need to remove defaultValue of Virtual Network and subnet in mainTemplate as well.

**Review by jkraj — APPROVED** [review]

File: main/mainTemplate.json (hunk 8 above)
Comment (andrei-maroz) [reply main/mainTemplate.json]: Done.

**Pull request merged**